Products/CMFPlone/browser/templates/basic_error_message.pt

```diff
@@ -10,6 +10,8 @@
 </head>
 
 <body>
+
+  <h1>${options/error_type}</h1>
```

Comment:
I don't think we should reveal this when the user is not a Manager. (I do see your comment that showing the traceback to managers is not working, and we should fix that...)

Reply:
The error_type is already in the html-title. That needs a condition then as well.